int32 robot_id
int32 start_x
int32 start_y
int32 end_x
int32 end_y